toolkits/springer/packages/springer-buttons/scss/50-components/buttons.scss

```diff
@@ -0,0 +1,15 @@
+.c-button {
+	@include font-size($button--font-size-normal)
```

Comment:
I don't see any value in using this `font-size()` mixin. Especially for small things like button.

Reply:
I'm happy to remove it - i was imagining it might be more apparent when next to other links that do size down but being a button it'll probably be ok.

Reply:
I think use of that mixin should be at app level anyway tbh